src/dispatch/case/flows.py

```diff
@@ -361,6 +361,16 @@ def case_update_flow(
         # we send the case updated notification
         update_conversation(case, db_session)
 
+    if case.has_channel and case.status != CaseStatus.closed:
+        # determine if case channel topic needs to be updated
+        if (
+            case.case_type.name != previous_case.case_type.name
+            or case.case_severity.name != previous_case.case_severity.name
+            or case.case_priority.name != previous_case.case_priority.name
+            or case.status != previous_case.status
+        ):
```

Comment:
I think this comparison is lengthy enough that it would be worth encapsulating it, purely for readability reasons. `e.g.`

```python
def case_details_changed(case: Case, previous_case: Case) -> bool:
    return (
            case.case_type.name != previous_case.case_type.name
            or case.case_severity.name != previous_case.case_severity.name
            or case.case_priority.name != previous_case.case_priority.name
            or case.status != previous_case.status
        )
```

Just a nit though.

Reply:
Good idea!